src/ContainerManager.ts

```diff
@@ -77,5 +90,33 @@ export class ContainerManager {
     if (style.position === "static") {
       container.style.position = "relative";
     }
+    if (this.options.autoResize) {
```

Comment:
@WoodNeck 
I think this feature is also needed in Flicking, how is it implemented?

@daybrush 
Don't you need this functionality for infiniteGrid as well? For example, in the case of relative infinite, which will be developed soon, I don't think grid is used. How is this functionality implemented in infiniteGrid, and is there any possibility of interference or problems with each other?

Reply:
@mixed 

I plan to disable Grid's autoResize and use InfiniteGrid's autoResize.

So I moved it to ContainerManager.

Reply:
@mixed It's already in Flicking